js/bybit.js

```diff
@@ -0,0 +1,1001 @@
+        //             "symbol": "BTCUSD",                 // symbol
+        //             "price": "9487",                    // price
+        //             "size": 336241,                     // size (in USD contracts)
+        //             "side": "Buy"                       // side
+        //         },
+        //         {
+        //             "symbol": "BTCUSD",                 // symbol
+        //             "price": "9487.5",                  // price
+        //             "size": 522147,                     // size (in USD contracts)
+        //             "side": "Sell"                      // side
+        //         }
+        //     ],
+        //     "time_now": "1567108756.834357"             // UTC timestamp
+        // }
+        const timestamp = this.safeFloat (response, 'time_now') * 1000;
+        const datetime = this.iso8601 (timestamp);
+        const result = {
+            'bids': [],
+            'asks': [],
+            'timestamp': timestamp,
+            'datetime': datetime,
+            'nonce': undefined,
+        };
+        // parseOrderBook can't be applied here because of specific structure of response
+        const data = this.safeValue (response, 'result', []);
+        if (data.length > 0) {
+            for (let i = 0; i < data.length; i++) {
+                const order = data[i];
+                const side = this.safeString (order, 'side');
+                let bidsOrAsks = undefined;
+                if (side === 'Sell') {
+                    bidsOrAsks = 'asks';
+                } else if (side === 'Buy') {
+                    bidsOrAsks = 'bids';
+                }
+                const amount = this.safeFloat (order, 'size');
+                const price = this.safeFloat (order, 'price');
+                if (bidsOrAsks !== undefined) {
+                    result[bidsOrAsks].push ([ price, amount ]);
+                }
+            }
+            result['bids'] = this.sortBy (result['bids'], 0, true);
+            result['asks'] = this.sortBy (result['asks'], 0);
+        }
+        return result;
+    }
+
+    async fetchBalance (params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        const response = await this.wapiGetPositionList (this.extend (request, params));
+        const retData = this.safeValue (response, 'result');
+        const result = { 'info': retData };
+        for (let i = 0; i < retData.length; i++) {
+            const position = retData[i];
+            const symbol = this.safeString (position, 'symbol');
+            const market = this.markets_by_id[symbol];
+            const currencyId = market['base'];
+            const code = this.safeCurrencyCode (currencyId);
+            const account = this.account ();
+            account['total'] = position['wallet_balance'];
+            account['used'] = position['position_margin'] + position['occ_closing_fee'] + position['occ_funding_fee'] + position['order_margin'];
```

Comment:
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings